File: 1Semestre/Introd Comp e Tec/aula3/ex3.py
# Desenvolva um programa em Python que solicite ao usuário digitar a idade de uma pessoa. 
# Após a digitação, o computador deverá verificar se esta pessoa é maior ou menor de idade e mostrar esta informação na tela do computador.
# Ainda verifique se a pessoa pode ou não votar.
# Exemplo:
# Se o usuário informar que a pessoa tem 17 anos, deverá ser mostrada na tela a mensagem “Pessoa menor de idade e pode votar”. 
# Se o usuário informar que a pessoa tem 18 anos, deverá ser mostrada na tela a mensagem “Pessoa maior de idade e pode votar”. 
# Caso a idade informada seja 15 anos a mensagem mostrada deve ser apenas “Pessoa menor de idade”.

idade = int(input("Digite a idade: "))

if idade >= 18:
    print("Maior de idade!")
    if idade <= 70:
        print("Você pode votar!")
    else:
        print("Você não pode votar!")
else:
    print("Menor de idade!")
    if idade >= 16:
        print("Você pode votar!")
    else:
        print("Você não pode votar!")

File: 1Semestre/Programacao de Computadores 1/aula2/ex3.c
// Desenvolva um programa que solicite ao usuário digitar dois valores inteiros. 
// Após a digitação, o computador deverá executar a multiplicação destes números e mostrar o resultado na tela do computador.
// Exemplo: se o usuário digitar os valores 11 e 13, o programa deve realizar a multiplicação destes números e mostrar na tela o resultado (143).

#include <stdio.h>

int main () {
  int n1,n2;

  printf("Digite o primeiro numero: ");
  scanf("%d", &n1);
  printf("Digite o segundo numero: ");
  scanf("%d", &n2);

  printf("%d", n1*n2);

  return 0;
}
